### PR TITLE
Cache Returns Types

### PR DESCRIPTION
**Pull request by MGatner**
**Description**
Adjusts a number of "mixed" return types to match their handler and User Guide specifications.

**Checklist:**
- [X] Securely signed commits
- [X] Component(s) with PHPdocs
- n/a Unit testing, with >80% coverage
- n/a User guide updated
- [X] Conforms to style guide

### CODE REVIEW

**Review by paulbalandan — COMMENTED** [review]
The handlers' docblocks should be adjusted too.

File: system/Cache/Handlers/WincacheHandler.php [hunk 1]
```diff
@@ -179,7 +179,7 @@ public function decrement(string $key, int $offset = 1)
 	 *
 	 * @return boolean
 	 *
-	 * @codeCoverageIgnore
+	 * @boolean Success or failure
```
Comment (paulbalandan) [inline system/Cache/Handlers/WincacheHandler.php:182]:
```suggestion
	 * @return boolean Success or failure
```

Comment (MGatner) [reply system/Cache/Handlers/WincacheHandler.php:182]: I pasted over the wrong docblock - return statement above. 😳 If you can fix this in a suggest I can merge it here (or you can) but I won't be on desktop for a while.

Comment (MGatner) [inline system/Cache/Handlers/WincacheHandler.php:182]:
```suggestion
 	 * @return boolean Success or failure
 	 *
	 * @codeCoverageIgnore
```

**Review by MGatner — COMMENTED** [review]
Nevermind, looks like I can make my own suggestions!

File: system/Cache/Handlers/WincacheHandler.php [hunk 2]
```diff
@@ -179,7 +179,7 @@ public function decrement(string $key, int $offset = 1)
 	 *
 	 * @return boolean
```
Comment (MGatner) [inline system/Cache/Handlers/WincacheHandler.php:180]:
```suggestion
 	 * @return boolean Success or failure
```

File: system/Cache/Handlers/WincacheHandler.php (hunk 1 above)
Comment (MGatner) [inline system/Cache/Handlers/WincacheHandler.php:182]:
```suggestion
	 * @codeCoverageIgnore
```

**Comment by MGatner** [comment]
Nevermind the nevermind, you cannot commit a suggestion over a deleted line. I will get this fixed later.

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**